ddtrace/profiling/collector/_memalloc_tb.c

```diff
@@ -180,6 +189,9 @@ traceback_to_tuple(traceback_t* tb)
         PyTuple_SET_ITEM(frame_tuple, 1, PyLong_FromUnsignedLong(frame->lineno));
         PyTuple_SET_ITEM(frame_tuple, 2, frame->name);
         Py_INCREF(frame->name);
+        /* Class name */
+        PyTuple_SET_ITEM(frame_tuple, 3, empty_string);
+        Py_INCREF(empty_string);
```

Comment:
We INCREF because the tuple stores it (and `SET_ITEM` does not incref itself). This is going to be decref when the tuple is not used anymore by the Python side. Same as `frame->name` above.